addons/waypoint/script_component.hpp

```diff
@@ -0,0 +1,15 @@
+#define COMPONENT waypoint
```

Comment:
Personally I think plural would be more in-place for component concerning (possibly) multiple waypoints.
```suggestion
#define COMPONENT waypoints
```

Reply:
I agree, do not forget to change pboprefix and folder names too.